lib/state_machines/machine.rb

```diff
@@ -658,6 +658,12 @@ def dynamic_initial_state?
     #   directly to the object
     def initialize_state(object, options = {})
       state = initial_state(object)
+
+      unless object.respond_to? attribute
+        object.class.send(:alias_method, attribute, :state)
```

Comment:
I don't think that's actually the issue here is it? `:state` is hardcoded in many places as the default column / attribute. Also I believe thats the behaviour expected by sm-ar. I'm more worried about the huge hack of this two lines. What you think about just dropping this feature for now? I'd ammend the commits here and drop the related tests in sm-ar.